op-monitorism/psp_executor/README.md

```diff
@@ -0,0 +1,24 @@
+### PSP Executor Service
```

Comment:
This code does not seems to be destined to be a monitoring feature, but executes actions.
What is the reason for adding this functionality in here?

Reply:
I do think concentrating our efforts in one centralized repository for detection and IR tooling is good things especially because the architecture of the PSP execution **service** is a service that will run with the same specification than the rest of the deamon we got inside K8s.